Jenkinsfile

```diff
@@ -33,9 +30,7 @@ pipeline {
         DEPLOY_SUCCESS = false
         // Determines if Elasticsearch image should be updated
         IS_ES_IMAGE_UPDATED = 'false'
-        // Determines if Cypress image should be updated
-        IS_CYPRESS_IMAGE_UPDATED = 'false'
-        DOCKER_HUB_REGISTRY = 'https://hub.docker.com'
+        DOCKER_HUB_REGISTRY = 'https://registry.hub.docker.com'
```

Comment:
I think perhaps we can remove this definition and use the default `docker login` below without a registry parameter.

Reply:
That doesn't work — see the build history of https://github.com/cfpb/consumerfinance.gov/pull/6498#partial-pull-merging